packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -328,25 +358,71 @@ export class VerifiedFetch {
       }
     }
 
+    /**
+     * range-request validation. This shouldn't be required, but unixfs.cat isn't throwing in some cases.
```

Comment:
Can you open an issue or better yet, a PR with some failing test cases for when `unixfs.cat` isn't throwing when it should?

Reply:
Yep. I wanted to make sure I was actually passing the right things.. i've got the test expectations and params cognizable now, so I'm sure something is off with .cat. will open issues when I can

Reply:
apparently it's working fine, it was a bug in my code :) surprised?! haha